src/herder/HerderImpl.cpp

```diff
@@ -764,6 +763,27 @@ HerderImpl::ledgerClosed(bool synchronous)
     }
 }
 
+void
+HerderImpl::ledgerClosed(bool synchronous)
```

Comment:
nit: (for future) - the diff would have made more sense if you defined `ledgerClosed` before `maybeTriggerNextLedger`